test/user.test.js

```diff
@@ -1406,19 +1407,21 @@ describe('User', function() {
 
   describe('Verification', function() {
     describe('user.verify(verifyOptions, options, cb)', function() {
+      const ctxOptions = {testFlag: true};
+      let verifyOptions;
+
+      beforeEach(function() {
+        // reset verifyOptions before each test
+        verifyOptions = {
+          type: 'email',
+          from: 'noreply@example.org',
+        };
+      });
+
       it('Verify a user\'s email address', function(done) {
         User.afterRemote('create', function(ctx, user, next) {
           assert(user, 'afterRemote should include result');
 
-          var verifyOptions = {
-            type: 'email',
-            to: user.email,
-            from: 'noreply@myapp.org',
-            redirect: '/',
-            protocol: ctx.req.protocol,
-            host: ctx.req.get('host'),
-          };
-
```

Comment:
refactoring existing tests to use the shared `verifyOptions`
we can simplify as most of these options are defaulted in `user.verify` implementation : `redirect`, `to`, `protocol`, `host`